.github/CODEOWNERS

```diff
@@ -1342,6 +1343,9 @@ x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai
 /.eslintignore @elastic/kibana-operations
 
 # Appex QA
+/x-pack/test/functional/config.*.js @elastic/appex-qa
+/x-pack/test/functional/config.*.ts @elastic/appex-qa
```

Comment:
is it possible to use `/x-pack/test/functional/config.*.*` or convert those few to `.ts`?

Reply:
Converting would be way outside of the scope and I've thousands more assignments to go through. 
But a better glob sounds great!

Reply:
https://github.com/elastic/kibana/pull/197253/commits/1f9ee0b395dc1d6aa207da5a537d5a0a7fa41e3a